torch/csrc/jit/codegen/cuda/kernel_ir.h

```diff
@@ -22,6 +22,38 @@ namespace jit {
 namespace fuser {
 namespace kir {
 
+#if 0
```

Comment:
Is there a reason to leave this in the PR?

Reply:
Good catch - no good reason. It's likely going to be back in the next iteration but I will remove it from this PR.